banking_system/user_management/utility/twofa.py

```diff
@@ -0,0 +1,43 @@
+import boto3
+import math
+import random
+from user_management.models import User, UserLogin
+
+
+def send_otp(otp, phone_number):
+    client = boto3.client(
+        "sns",
+        aws_access_key_id="",
+        aws_secret_access_key="",
+        region_name="eu-west-1"
+    )
+    msg = 'Your otp for login is {0}'.format(otp)
+    phone_number = '+1' + str(phone_number)
+    print('msg', msg)
+    print('phone_number', phone_number)
+    client.publish(
+        PhoneNumber=phone_number,
+        Message=msg
+    )
```

Comment:
I still think it would be nice to do this logic either through email or a phone number service that doesn't charge. Twilio used to not charge but add a "Sent through Twilio API:" type message which would be fine for our purposes. IDK if that is the case anymore. I won't block PR for this though. Just recording thoughts for later.

Reply:
I will look at it.